app/services/quotes_service.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class QuotesService
+  def self.quotes(keyword)
+    parse(conn.get("/search/quotes?query=#{keyword}&limit=10"))
+  end
+
+  def self.conn
+    conn ||= Faraday.new('https://api.quotable.io/')
+  end
+
+  def self.parse(response)
+    JSON.parse(response.body, symbolize_names: true)
+  end
+end
```

Comment:
👍 At this time conn will only ever get called once with each http request. So, it's not necessary to memoize, but it's a good practice to get in as soon as you add another endpoint to this service.